test/integration/update-community-controller.spec.ts

```diff
@@ -234,6 +242,36 @@ test('Update Community Controller', async function ({ components, stubComponents
               expect(body.message).toBe('Community updated successfully')
             })
 
+            describe('and the communiy is private', () => {
+              beforeEach(async () => {
+                await components.communitiesDb.updateCommunity(communityId, {
+                  private: true
+                })
+              })
+
+              afterEach(async () => {
+                await components.communitiesDb.updateCommunity(communityId, {
+                  private: false
+                })
+              })
+
+              it('should not switch the community to public', async () => {
```

Comment:
(I'm taking into consideration that this test is executed under the context that a description exists)
As there are multiple variations of the properties that can be updated, it might be better in terms of maintenance to check these variations with unit tests. We could focus on cases that test that each property was properly updated.